netflix/archaius/src/test/java/org/wildfly/swarm/ts/netflix/archaius/ArchaiusTest.java

```diff
@@ -0,0 +1,38 @@
+package org.wildfly.swarm.ts.netflix.archaius;
+
+import org.apache.http.client.fluent.Executor;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.impl.client.BasicCookieStore;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.jaxrs.JAXRSArchive;
+import org.wildfly.swarm.ts.netflix.archaius.ArchaiusResource;
+import org.wildfly.swarm.ts.netflix.archaius.RestApplication;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+
+@RunWith(Arquillian.class)
+public class ArchaiusTest {
+
+    @Deployment(testable = false)
+    public static Archive<?> deployment() {
+        JAXRSArchive archive = ShrinkWrap.create(JAXRSArchive.class);
+        archive.addClass(ArchaiusResource.class).addClass(RestApplication.class);
+        archive.addAsWebInfResource(new File("src/main/resources/project-defaults.yml"));
+        return archive;
+    }
+
+    @Test
+    public void test() throws Exception {
+        Executor executor = Executor.newInstance().use(new BasicCookieStore());
+
+        String response = executor.execute(Request.Get("http://localhost:8080/")).returnContent().asString();
+        assertThat(response).isEqualTo("50");
```

Comment:
I'd expect 77 here :-)

Reply:
Yeah, that's the reason I closed the pull :D

Reply:
Ah so it's returning 50 now? Weird.

Reply:
Yeah,  that is the default value from fraction. Probably it isn't getting the yaml file for some reason. Will investigate that on thursday...

Reply:
OK cool.